tests/test_converters/test_obo_graph_to_fhir.py

```diff
@@ -10,28 +14,85 @@
 from tests import IMBO, INPUT_DIR, NUCLEUS, OUTPUT_DIR
 from tests.test_implementations import ComplianceTester
 
-ONT = INPUT_DIR / "go-nucleus.json"
-OUT = OUTPUT_DIR / "go-nucleus.fhir.codesystem.json"
+
+DOWNLOAD_TESTS_ON = False
```

Comment:
@cmungall I think these ontologies I'm downloading average about 200mb. I figured it would not be desirable to slow down the test suite in general by doing these downloads all the time, so I added `DOWNLOAD_TESTS_ON = False`. So, the purpose of these tests is to run on an ad hoc basis.

I could commit a subset of these ontologies and that way they can always be run, but I want to focus first on getting some more content before the Connectathon first if that's alright.

Reply:
I think this is fine for now

In general it's better to have the control be via environment variable

We need better solutions in general in OAK for integration tests